package-list.json

```diff
@@ -1,22 +1,22 @@
 {
   "package-id": "hl7.fhir.uv.smart-web-messaging",
-  "title": "SMART Web Messaging Implementation Guide",
+  "title": "SMART Web Messaging",
   "canonical": "https://hl7.org/fhir/uv/smart-web-messaging",
+  "introduction": "This IG provides detailed guidance that helps implementers to use SMART Web Messaging to enable tight UI integration between EHRs and embedded SMART apps via HTML5’s Web Messaging. SMART Web Messaging allows applications to push unsigned orders, note snippets, risk scores, or UI suggestions directly to the clinician’s EHR session. Built on the browser’s javascript window.postMessage function, SMART Web Messaging is a simple, native API for health apps embedded within the user’s workflow."
   "list": [
     {
       "version": "current",
-      "date": "n/a",
       "desc": "Continuous Integration Build (latest in version control)",
       "path": "http://build.fhir.org/ig/HL7/smart-web-messaging",
       "status": "ci-build"
     },
     {
       "version": "1.0.0",
-      "date": "2022-03-08",
+      "date": "14-03-2022",
```

Comment:
Is this date intentional? https://confluence.hl7.org/display/FHIR/FHIR+IG+PackageList+doco says:

>date: the date at which the publication was made [yyyy-mm-dd]